src/prommis/evaporation_pond/evaporation_pond.py

```diff
@@ -0,0 +1,605 @@
+
+Degrees of Freedom
+------------------
+
+Evaporation Pond models have three degrees of freedom in addition to the feed stream state. The most common
+variables to specify are:
+
+* the surface area of the evaporation pond, ``surface_area``,
+* the average depth of liquid in the pond, ``average_pond_depth``, and
+* the rate of evaporation in units of length/time (or volume evaporated  per unit area per time), ``evaporation_rate``.
+
+Model Structure
+---------------
+
+The Evaporation Pond unit model does not use Control Volumes, and writes custom material balances and
+reaction constraints. The Evaporation Pond model has one inlet and one outlet ``Port`` (named
+``inlet`` and ``outlet`` respectively).
+
+Variables
+---------
+
+Evaporation Pond models have the following Variables.
+
+================= =================== =====================
+Variable          Name                Indexing Set(s)
+================= =================== =====================
+:math:`A_{t}`     surface_area        time
+:math:`D_{t}`     average_pond_depth  time
+:math:`V_{t}`     volume              time
+:math:`e_{t}`     evaporation_rate    time
+:math:`W_{t}`     water_loss_rate     time
+:math:`P_{t, j}`  precipitation_rate  time, component list
+:math:`X_{t, r}`  reaction_extent     time, reaction list
+================= =================== =====================
+
+Params
+------
+
+Evaporation Pond models have the following Params, which are used in the complementarity constraint
+for precipitation.
+
+================== ======== ============== =================================================================
+Parameter          Name     Indexing Set   Notes
+================== ======== ============== =================================================================
+:math:`\epsilon`   eps                     Smoothing parameter for smooth_max
+:math:`norm_{r}`   s_norm   reaction list  Normalizing factor, should match magnitude of solubility product
+:math:`scale_{r}`  s_scale  reaction list  Scaling factor for reaction product, Q = Ksp - f(C)
+================== ======== ============== =================================================================
+
+Constraints
+-----------
+
+Evaporation Pond models have the following constraints.
+
+Component balances:
+
+For the solvent:
+
+.. math:: 0 = F_{in,t,j} - F_{out,t,j} + P_{t_j} + W_{t}
+
+For other species:
+
+.. math:: 0 = F_{in,t,j} - F_{out,t,j} + P_{t_j}
+
+where :math:`F_{in,t,j}` and :math:`F_{out,t,j}` are the flow rate of component :math:`j` in to and out of
+the pond at time :math:`t`.
+
+Stoichiometry Constraint:
+
+.. math:: P_{t, j} = \sum_r{n_{r,j} \times X_{t,r}}
+
+where :math:`n_{r,j}` is the stoichiometric coefficient for component :math:`j` in reaction :math:`r`.
+
+Equilibrium Constraint:
+
+.. math:: Q_{t, r} - \max{0, Q_{t,r}-\bar{X}_{t,r}} = 0
+
+where :math:`Q_{t,r} = \ln{K_{t,r} - \sum_j{-n_{r,j} \times \ln{C_{t,j}} and :math:`K_{t,r}` is the
```

Comment:
```suggestion
where :math:`Q_{t,r} = \ln{K_{t,r}} - \sum_j{-n_{r,j} \times \ln{C_{t,j}}` and :math:`K_{t,r}` is the
```
this might resolve this issue, missing end brace. @andrewlee94 please check that this correctly groups the terms.